src/engine/database/neo4j.rs

```diff
@@ -30,15 +30,20 @@ use tokio::runtime::Runtime;
 /// # use warpgrapher::engine::database::neo4j::Neo4jEndpoint;
 /// #
 /// # fn main() -> Result<(), Box<dyn std::error::Error>> {
-///     let ne = Neo4jEndpoint::from_env()?;
+///     let ne = Neo4jEndpoint {
+///         host: "127.0.0.1".to_string(),
+///         port: 7687,
+///         user: "neo4j".to_string(),
+///         pass: "password".to_string()
+///     };
 /// #    Ok(())
 /// # }
 /// ```
 pub struct Neo4jEndpoint {
-    host: String,
-    port: u16,
-    user: String,
-    pass: String,
+    pub host: String,
```

Comment:
We may have talked about this, so apologies if I've forgotten, but what was the use case / rationale for making these public?

Reply:
Need to instantiate an database endpoint from a source other than environment variables.

Reply:
Why does that require making the fields of the struct public?  Wouldn't that work as long as you have a `Neo4JEndpoint::new` function?

Reply:
Done.

Reply:
Fixed.